packages/react-router/tests/fileRoute.test-d.tsx

```diff
@@ -17,7 +17,7 @@ const postRoute = createFileRoute('/_postLayout/posts/$postId_')()
 
 const protectedRoute = createFileRoute('/(auth)/protected')()
 
-declare module '../src/fileRoute' {
+declare module '@tanstack/router-core' {
```

Comment:
why don't we move the type test to router-core as well?

Reply:
Maybe we can in another pr. atm I don't want to change them too much because I want to rely on them to check I have broken anything

Reply:
no worries, just trying to understand where you are heading